partials/eligibility.html

```diff
@@ -0,0 +1,102 @@
+<div id=Credits class="credits fixed-center-column"
+                        ng-show="showReport()" ng-cloak>
+  <h2 id="nav-credits">Retirement Benefits Eligibility</h2>
+
+  <p>
+    You must earn 40 <a href="https://www.ssa.gov/planners/credits.html">credits</a> to be eligible for for a normal 
+    retirement benefit from your own work history.  You can earn a maximum of four credits each year.
```

Comment:
Maybe use `4` instead of `four` here, for consistency. You use numerical representations of the other numbers in this block of text.

It also might be useful to underline `maximum of 4 credits each year` or something. I could see someone looking at the table and scratching their head when (earnings / earnings per credit) is obviously way more than 4.

Reply:
Fixed.